src/app/thank-you/account-benefits/account-benefits.tpl.html

```diff
@@ -0,0 +1,15 @@
+<div class="panel">
+  <div class="panel-body">
+    <h3 class="panel-name">Register Your Account for Online Access</h3>
+    <p>Benefits of registering for an account include:</p>
+    <ul>
+      <li>Save time during checkout</li>
+      <li>Update your contact info</li>
+      <li>View your donation history</li>
+      <li>Manage your recurring gifts</li>
+      <li>Change your bank account or credit card information</li>
+    </ul>
+    <a data-toggle="modal" data-target="#signupModal" class="btn btn-primary btn-block">Register Your Account</a>
+    <a data-toggle="modal" data-target="#signinModalRec">Register (User Match Alternate Modal)</a>
```

Comment:
For now, remove this one and link the button to show the register modal from sessionModal.service.